content/soarian/dstu2/care-provision/careplan.md

```diff
@@ -0,0 +1,96 @@
+---
+title: CarePlan | Soarian DSTU 2 API
+---
+ 
+# CarePlan
+
+* TOC
+{:toc}
+
+## Overview
+
+For the CarePlan resource query, the API returns the most current charted data applicable to encounter(s) contained by the explicit or implicit date(s) in the query for content configured by the Soarian Clinicals client as relevant to this resource.  The ONC (Office of the National Coordinator) CCDS (common clinical data set) definition is defined to include types of data relevant to this resource: Concerns, Goals, Assessments, and Plans.  Note that concerns may or may not be classified as problems or diagnosis and concerns and goals may be patient and/or caregiver expressed.  While conceptually health concerns, goals, planned activities and assessments are four different data sets, in clinical practice they maybe documented as a consolidated thought and an individual query of just one of the data types may result in clinical inaccuracy. Additionally, clients may configure variable amounts of structured charting.  For example, the same clinical scenario might be documented in practice as: 
```

Comment:
Can the care plan have any structure to it, outside of the narrative? IE: relatedPlan, goal, participant or activity fields can be filled out? This paragraph makes that sound possible, but none of the examples show this.